### ISSUE TITLE
Accessibility broken down on Admin Client

### ISSUE THREAD

**Issue by monatis** [TryGhost/Ghost#9489]
The publish button on the New Post screen (admin client) is no longer accessible to screen readers. As a rule, focus modes of UI elements should never be disabled or overridden. However, the latest update to this button has made it inaccessible to screen readers users interacting UI with a keyboard, making Ghost completely unusable for them. I tried to read javascript code to figure out the function that triggers a click on this button, but it turns out to be useless in the end.

Is there any consideration or planned update on this?

**Comment by monatis** [comment]
I figured out a workaround: I simply run `javascript:void $('button:contains("Publish")').trigger('click');` and the post gets published. But this still requires a solution, as the buttons inside the dropdown menu are not accessible.

**Comment by kevinansfield** [comment]
Hey @monatis 👋 Thank you for bringing this to our attention. The publish menu is due for a re-design soon and is being tracked internally, we'll be sure to address the accessibility issues as part of that 😄

**Comment by monatis** [comment]
Thanks @kevinansfield. I'm always ready for testing or any other contribution to this.
